app/config/sources.inc.php

```diff
@@ -894,6 +894,12 @@
         'deadline'          => '2017-06-19',
         'supported_locales' => ['de'],
     ],
+    'snippets/2017/aug2017.lang' => [
```

Comment:
I have the feeling this name choice is going to bite you in a couple of weeks, since this sounds like an out of cycle request 🐕